src/Resources/views/standard_layout.html.twig

```diff
@@ -118,32 +118,33 @@ file that was distributed with this source code.
                     </noscript>
                 {% endblock %}
                 {% block sonata_nav %}
-                    <ul class="navbar-nav">
-                        <li class="nav-item">
-                            <a class="nav-link" data-widget="pushmenu" href="#">
-                                <span class="sr-only">{{ 'toggle_navigation'|trans({}, 'SonataAdminBundle') }}</span>
-                                <i class="fas fa-bars"></i>
-                            </a>
-                        </li>
-                    </ul>
-
-                    <div class="navbar-left">
-                        {% block sonata_breadcrumb %}
-                            <div class="hidden-xs">
-                                {% if _breadcrumb is not empty or action is defined %}
-                                    <ol class="nav navbar-top-links breadcrumb">
+                    <di class="d-flex flex-row">
```

Comment:
this di looks strange

Reply:
Hey @jordisala1991 why is it looks strange?

I was doing it, getting [adminlet 3](https://adminlte.io/themes/v3/) as example, and the `navbar` and `breadcrumbs`, changed a bit.

Reply:
after fix the pipelines I will take a look again and check exactly why I needed to change this like this ;)

Reply:
I thought it was a div not a di.

Reply:
ops hahahah, true :D